### DISCUSSION TITLE
Logging-in as another user > Provide link at top of screen to log back as admin

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#8138] [Ideas]
> Created by: Jeremy Abrahams (jeremy@mightyinthemidwest.com) on 2015/06/12 19:36:26 +0000
> Votes at time of UserVoice import: 13

-----------------------------------

As an admin I would like to be able to quickly switch back and forth between different user types. But, I'd like to have the ability to revert back to an admin without having to logout and then log back in.

**Comment by angrybrad** [comment]
> Posted by Jeremy Abrahams (jeremy@mightyinthemidwest.com) on 2015/06/12 19:36:26 +0000
-----------------------------------

Something like a top-bar would be nice.

**Comment by jasonmccallister** [comment]
Was just going to leave this comment/fr. This would be great to test permissions!
